hub/package-manager/winget/index.md

```diff
@@ -36,11 +36,14 @@ Windows App Installer includes the production version of the winget tool. To try
 To install the stable release of winget on Windows Sandbox, follow these steps from a Windows PowerShell command prompt:
 
 ```powershell
-$ProgressPreference='Silent'
-Invoke-WebRequest -Uri https://github.com/microsoft/winget-cli/releases/download/v1.3.2691/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle -OutFile .\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle
```

Comment:
Would this work with https://aka.ms/getwinget ? That way we don't necessarily have to update this with every release.

Reply:
It does work, and I am happy to change that, however given the dependencies changed between 1.3 and 1.4, if that happens again then this will still break. I don't know what is more likely, so happy to be guided

Reply:
We will likely have to manage the dependencies in future changes. We're working on PowerShell DSC Resources that also contain the scripts to achieve this in a more supportable way in the future: 
* https://github.com/microsoft/winget-cli/issues/2697

Reply:
Understood, shall I change the uri to https://aka.ms/getwinget then?

By the way, I personally prefer this PowerShell script way of installing winget into a running Sandbox, to the alternative of updating the sandbox disk image